README.md

```diff
@@ -1 +1,11 @@
-# odoo-sotto_sale_payment_status
\ No newline at end of file
+# Odoo Sale Payment Status Add-on (sotto_sale_payment_status)
+
+This add-on aims at providing payment status information on Odoo sale orders. For more information please visit official add-on website/Odoo platform [here](https://apps.odoo.com/apps/modules/14.0/sotto_sale_payment_status/#).
+
+This documentation outlines the modifications made to the "sotto_sale_payment_status" addon in the Odoo platform.
+
+Release notes:
+
+## Version 14.0.1.0.0
+* Change Description: Modification: Changed the value of the auto_install attribute to True in the `__manifest__.py`file.
```

Comment:
> Changed the value of the `auto_install` attribute to `True`...

Reply:
Should be good now, I've just realised you were referring to the code formatting.